resources/assets/components/pages/VoterRegistrationDrivePage/Beta/HeroSection.js

```diff
@@ -18,6 +19,25 @@ const HeroSection = ({ user, campaignInfo, modalToggle }) => {
    * TODO: Check for a voting-reasons query parameter, and render values in quote if present.
    * @see https://www.pivotaltracker.com/story/show/172087475
    */
+
+  const formatEndofBetaCauseSentence = () => {
+    const userCauses = query('voting-reasons').split(', ');
```

Comment:
Will the query value be divided by a comma _and_ space here?

Reply:
Also, we'll probably need to use a lodsh `get` here to ensure we don't call `split` on an `undefined` value
```suggestion
    const userCauses = get(query('voting-reasons'), '').split(', ');
```

Then we could even remove the first condition directly below since we're guaranteed an (at least empty) array!

Reply:
Ohhh and also, I wonder if we should whitelist the accepted causes and filter out anything else before we process the array to avoid any hardcoded naughty values?

Reply:
If a voting-reason / cause doesn't have a corresponding key in our config, it should not be displayed - so we don't have to worry about users customizing their `voting-reasons` values. Refs https://github.com/DoSomething/phoenix-next/pull/2126#pullrequestreview-409558496

Reply:
Thanks for the feedback. I pushed the revised changes.

Currently, the code works for all test cases, except when there are no elements in `voter-reasons` param. I tried @mendelB suggestion for using `get` before splitting the array but it didn't work. It returns `Cannot read property 'split' of undefined` which makes sense. I also tried to check with a simple `if(userCauses && userCauses.length)` before splitting the array but it breaks the previously passing cases 
cc: @aaronschachter